flux_led/__init__.py

```diff
@@ -661,7 +661,8 @@ def setRgb(self, r,g,b, persist=True, brightness=None):
         msg.append(int(g))
         msg.append(int(b))
         msg.append(0x00)
-        msg.append(0xf0)
+        msg.append(0x00)
+        msg.append(0x00)
```

Comment:
@simpat1zq thanks for your contribution. Is there any specific reason to have 3 sequential `msg.append(0x00)` entries?